### PR TITLE
[feat] use the IPFS proxy

### LINKED ISSUE #39: [bug] interactive OBJKTs are not vertically aligned when using fullscreen mode on the minting preview page
**Describe the bug**
Interactive OBJKTs are not vertically aligned when using fullscreen mode on the minting preview page. The vertical alignment is correct in fullscreen mode once the OBJKT is minted.

**To Reproduce**
1. go to mint page
2. upload an interactive OBJKT
3. go to preview page
4. open full screen mode (the issue is more obvious on larger monitors)

**Expected behavior**
The OBJKT should appear vertically centered

**Screenshots**
image
Preview

image
Minted

---

### Tasks:

- [x] Fix the preview page for interactive tokens in fullscreen.
- [ ] Vertically center non full width/height tokens:
    As mentioned by formless on discord
    image
    https://teia.art/objkt/670193 - Shifted
    https://versum.xyz/token/versum/18814 - Working

### LINKED ISSUE #149: [bug] .md files can't be previewed for minting
### Describe the bug

md files cannot be previewed

### Reproduction

Upload a .md text file for minting.
Press the preview button
Look at the browser console for an error.

Screenshot from 2022-08-04 15-04-05

### Expected behavior

See md file contents on preview

### Platform and versions

```shell
Ubuntu
Google Chrome
```

### Console output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by melMass**

### CODE REVIEW

**Comment by melMass** [comment]
~~As discussed on Discord, CID versions might be an issue 😫~~, 

NFT.storage's CID has a toV0() method.

image

File: src/data/ipfs.js [hunk 1]
```diff
@@ -308,5 +307,10 @@ async function uploadMetadataFile({
     metadata.contentRating = contentRating
   }
 
+  if (rights === 'custom') {
+    metadata.rightsUri = rightUri
```
Comment (Zir0h) [inline src/data/ipfs.js]: rightsUri or rightUri? I get confused :sweat_smile: 
// edit: sorry, didn't see the "not ready for review yet" msg

Comment (melMass) [reply src/data/ipfs.js]: > rightsUri or rightUri? I get confused 😅

Hehe good point, we should match them all, on the indexer it's using what for now?

Comment (Zir0h) [reply src/data/ipfs.js]: The indexer uses `rightUri`

**Comment by melMass** [comment]
Note for later:

- Paths for subfolders are "screwed" in the utils/html zip / unzip phases.
- Mimetypes is defined too many times (not a big deal)

**Pull request merged**